src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+public class Task {
+    private boolean isCompleted = false;
+    private String taskDescription;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+    }
+
+    public void setTaskAsDone() {
+        this.isCompleted = true;
+    }
+
+    public boolean isCompleted() {
+        return this.isCompleted;
+    }
+
+    public String getTaskDescription() {
+        return this.taskDescription;
+    }
+
+    public void printAddTaskMessage() {
+        System.out.println("Got it. I've added this task: ");
+        System.out.println("  " + this.toString());
+    }
+
+    public void printMarkTaskAsDone() {
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("  " + this.toString());
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Since there are different characters (e.g. "X" or " ")  that could be assigned to `taskCompleted`, you could provide [javadocs comments ](https://se-education.org/guides/conventions/java/basic.html#comments)to explain the method is doing in general as these characters seem arbitrary.